openstack-provisioner/hooks/ansible/post/post.yaml

```diff
@@ -0,0 +1,327 @@
+            ip route add default via {{ ansible_default_ipv4.gateway }} dev eth0 table mgmt_rt
+            ip rule add from {{ ansible_default_ipv4.address }}/32 table mgmt_rt
+            ip rule add to {{ ansible_default_ipv4.address }}/32 table mgmt_rt
+        become: true
+
+      - name: Add hostname to /etc/hosts
+        lineinfile:
+            path: /etc/hosts
+            line: "{{ ansible_default_ipv4.address }} {{ ansible_hostname }} {{ ansible_fqdn }}"
+        become: true
+
+      - name: Create swap
+        shell: |
+            dd if=/dev/zero of=/swapfile bs=4096 count=1048576
+            chmod 600 /swapfile
+            mkswap /swapfile
+            swapon /swapfile
+        become: true
+
+      - name: Generate nodes json
+        vars:
+            bmc_start_port: 6230
+        template:
+          src: "nodes.json.j2"
+          dest: /tmp/master_nodes.json
+
+      - name: check existing keypair
+        stat:
+          path: "~/.ssh/id_rsa.pub"
+        register: keypair_result
+
+      - name: generate keypair if not present
+        shell: "ssh-keygen -f ~/.ssh/id_rsa -t rsa -N ''"
+        when: keypair_result.stat.exists == False
+
+      - name: register public key
+        shell: "cat ~/.ssh/id_rsa.pub"
+        register: provision_key
+
+      - name: clear authorized_keys on router
+        become: true
+        shell: "> /root/.ssh/authorized_keys"
+        delegate_to: routerbmc-0
+
+      - name: transfer public key to router
+        become: true
+        copy:
+          dest: /root/.ssh/authorized_keys
+          content: "{{ provision_key.stdout }}"
+        delegate_to: routerbmc-0
+
+      - name: bring up network interfaces
+        shell: |
+            ip link set dev {{ item }} up
+            ip link set dev {{ item }} mtu 1450
+        become: true
+        with_items:
+            - 'eth1'
+            - 'eth2'
+
+      - name: Install git
+        become: true
+        package:
+            name: git
+            state: installed
+
+      - name: Sync local dev-scripts to provisioning instance
+        synchronize:
+            src: "{{ hostvars['localhost']['playbook_dir'] }}/../../../../"
+            dest: "~/dev-scripts/"
+
+      - name: Copy config
+        shell: "cp config_example.sh config_{{ ansible_user }}.sh"
+        args:
+          chdir: "~/dev-scripts"
+
+      - name: Configure secret
+        replace:
+            path: "~/dev-scripts/config_{{ansible_user}}.sh"
+            regexp: "PULL_SECRET=''"
+            replace: "PULL_SECRET='{{ openshift_secret }}'"
+
+      - name: Set DEPLOY_OVB
+        lineinfile:
+            path: "~/dev-scripts/config_{{ansible_user}}.sh"
+            line: "DEPLOY_OVB=true"
+
+      - name: Run 01_install_requirements.sh
+        shell: "bash 01_install_requirements.sh &> 01_install_requirements.log"
```

Comment:
what's the reason to hide log messages from the operator? Perhaps we could pipe it through `tee` both to capture logs on disk AND to show them to the operator?